### PR TITLE
Graphical cc

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

**Comment by mvidner** [comment]
Thanks, but it is not at all apparent whether this is right or wrong. Will it even fix the reported problem?

**Comment by jreidinger** [comment]
@mvidner this change adds dependency, that if control-center-gnome and yast2-packager is installed, it will install also this package, so as mentioned in bug report, it will report qt packager instead of ncurses one, which is problematic for graphical control center.

File: package/libyui-qt-pkg.spec [hunk 1]
```diff
@@ -58,6 +58,8 @@ Obsoletes:      yast2-qt-pkg < 2.42.0
 
 Provides:       libyui_pkg
 Supplements:    packageand(libyui-qt:yast2-packager)
+# control center gnome requires graphical pkg and gtk is dropped (bnc#999031)
+Supplements:    packageand(yast2-control-center-gnome:yast2-packager)
```
Comment (mvidner) [inline package/libyui-qt-pkg.spec]: I think that if the already existing `packageand(libyui-qt:yast2-packager)` has not helped then neither will this one. Have you tested the fix? I haven't seen the logs from the bug yet.

File: package/libyui-qt-pkg.changes [hunk 2]
```diff
@@ -18,6 +25,7 @@ Mon Oct 10 14:04:41 UTC 2016 - jreidinger@suse.com
 
 - Improve message shown when user want to quit without saving
   changes (bsc#849084)
+>>>>>>> origin/master
```
Comment (mvidner) [inline package/libyui-qt-pkg.changes]: Psst mister, wanna buy a conflict marker?

Comment (jreidinger) [inline package/libyui-qt-pkg.changes]: it is for free :wink: 
fixed

**Pull request merged**